web/src/components/languageselect.jsx

```diff
@@ -43,6 +43,12 @@ const LanguageSelectContainer = styled.ul`
     }
     &.preferred-language {
       background-color: rgba(0, 0, 0, 0.1);
+      position: absolute;
+      top: 0;
+    }
+    &.other-language {
+      position: relative;
+      top: 30px;
```

Comment:
For some reason this produce a small gap, my local testing found that 27px would remove the gap while still not overlapping.

Reply:
You are right.
27px removes the unnecessary gap.